### ISSUE TITLE
difference between bop version and paper version for YCB-V dataset

### ISSUE THREAD

**Issue by RuyiLian** [suyz526/ZebraPose#6]
Hi,

Thanks again for sharing your great work! I am a bit confused about the bop version and paper version of YCB-V dataset. The configs looks the same, except the option `bop_challange`. Does `bop_challange` only affect the test data? Thanks!

**Comment by suyz526** [comment]
> Hi,
> 
> Thanks again for sharing your great work! I am a bit confused about the bop version and paper version of YCB-V dataset. The configs looks the same, except the option `bop_challange`. Does `bop_challange` only affect the test data? Thanks!

Hi, 

you are right, `bop_challange` only affects the test data. BOP challenge only uses a subset from the test set.

There is another difference. For the experiments in the paper, we used `crop_resize`. This type of resizing will directly resize the detected bounding box to 256x256. 

For BOP challenge, we used `crop_square_resize`. Here we pad the detected bounding box to be a square, and then resize it to 256x256.
